mailpoet/views/parsley-translations.html

```
@@ -1,27 +1,34 @@
 
+<#
+  Each message is emitted with json_encode(), which supplies its own quotes and escapes
+  anything that would otherwise end the JavaScript string. Interpolating a translation
+  straight into '...' breaks on the first apostrophe: the script then fails to parse and
+  every message here is silently lost, so Parsley falls back to its built-in English.
+  Most languages need an apostrophe somewhere, so this has to stay json_encode().
+#>
 Parsley.addMessages('mailpoet', {
-  defaultMessage: '<%= __("This value seems to be invalid.") %>',
+  defaultMessage: <%= json_encode(__("This value seems to be invalid.")) %>,
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)